popdat/src/lib.rs

```diff
@@ -20,8 +20,12 @@
 #[macro_use]
 extern crate log;
 
+#[macro_use]
```

Comment:
While I'm nitpicking style, maybe we should sort this above the `log` too. Although honestly if rustfmt doesn't do it, stuff like this is just busywork. :\

Reply:
Is it that you'd prefer to keep the alphabetical within the sections? That sounds fine - just making sure I understand.

Reply:
Yeah, I think that's the general idea. With `macro_use` I guess it's not a hard-and-fast rule, because I think the order of these can have a real effect if there are interdependencies.